web/modules/custom/pineapple_mail/pineapple_mail.module

```diff
@@ -0,0 +1,61 @@
+<?php
+
+/**
+ * @file
+ * Contains pineapple_mail.module.
+ */
+
+use Drupal\Component\Utility\Html;
+
+/**
+ * Implements hook_mail().
+ */
+function pineapple_mail_mail($key, &$message, $params) {
+  $options = array(
+    'langcode' => $message['langcode'],
+  );
+  switch ($key) {
+    case 'node_insert':
+      $message['from'] = \Drupal::config('system.site')->get('mail');
+      $message['subject'] = t('Subject: @title', array('@title' => $params['title']), $options);
+      $message['body'][] = Html::escape($params['message']);
+      break;
+  }
+}
+
+function pineapple_mail_ready_to_review(Drupal\Core\Entity\EntityInterface $entity) {
```

Comment:
Name this function to something more general so that it can be reused in the future if needs be.

Reply:
Is that name okay? I had to keep the first mail because that's the module name.